source/nodejs/adaptivecards/src/table.ts

```diff
@@ -0,0 +1,586 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { CardElement, StylableCardElementContainer, SerializationContext, Container, ContainerStyleProperty } from "./card-elements";
+import { HorizontalAlignment, VerticalAlignment, SizeUnit, ValidationEvent, TypeErrorType, Spacing } from "./enums";
+import { TextStyleDefinition } from "./host-config";
+import { BaseSerializationContext, BoolProperty, CustomProperty, EnumProperty, property, PropertyBag,
+    PropertyDefinition, SerializableObject, SerializableObjectCollectionProperty, Versions } from "./serialization";
+import { SizeAndUnit, PaddingDefinition } from "./shared";
+import { Strings } from "./strings";
+import { stringToCssColor } from "./utils";
+
+export class ColumnDefinition extends SerializableObject {
```

Comment:
Feel like this should be `TableColumnDefinition` to prevent confusion with `ColumnSet` `Column`s

Reply:
I don't really mind, but this is per spec.

Reply:
If you feel strongly about this one, please send a mail to the team to suggest this name change which would also have to be reflected in the spec that Risheek has a PR for here: https://github.com/microsoft/AdaptiveCards/pull/5806